### PR TITLE
Azurenpmautoupdate

### PR DESCRIPTION
**Pull request by paulgmiller**

### CODE REVIEW

**Comment by paulgmiller** [comment]
Problems: 
1) it's ugly
2) I'm filtereing out pre-releases but other things could get in.
3) ordering isn't really garunteed
4) if we got more than 5 of ahead of the toggle we'd lose caching

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -200,13 +200,10 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
...
-AZURE_NPM_VERSIONS="
-1.1.2
-1.1.0
-1.0.33
-1.0.32
-1.0.30
-"
+#Get the last 5 versions from azure-npm
+#This scipt sure isn't pretty so open to sugestestiosn to make it more elegant
+AZURE_NPM_VERSIONS=$(curl -s -L  https://mcr.microsoft.com/v2/containernetworking/azure-npm/tags/list | jq --raw-output '.tags | map(select(contains("-")!=true))[-5:] | .[] ')
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:205]: If you really want to get some automatic caching, maybe add this on top of the existing static list and adjust the static list periodically but much less frequently

**Comment by alexeldeib** [comment]
@paulgmiller doing housekeeping, want to revive this or close it?

**Comment by paulgmiller** [comment]
two years old giving up.

**Pull request closed without merging**